### DISCUSSION TITLE
next.js renders blank page or partially rendered page

### DISCUSSION THREAD

**Discussion by ravitdx** [vercel/next.js#13752] [Help]
Hi,
I am using nextjs for my website dynamic pages. I am using Apollo graphql to fetch data. Everything works correctly. But, occasionally, for some people frequently, sees a blank page or partially rendered page or non-interact able page. Is this expected behavior? Can I do something to prevent this?

I am using 
Node 12.x
next: ^9.2.2
@apollo/react-hooks: "^3.1.3,
@apollo/react-ssr: "^3.1.3

**Comment by todortotev** [comment]
Please use the issue template so we can help you.

**Comment by ravitdx** [comment]
Hi, 
I just want know whether its a common issue that might happen with dynamic content. It happens occasionally for  same page and same content (obviously same code). Is this because of slow rendering. Should we have to increase our server's CPU power.
